packages/example-contracts/lib/shared/deploy.helpers.ts

```diff
@@ -84,3 +86,13 @@ export const verifyContract = (
 
   execSync(command);
 };
+
+export const deployZetaTokenConsumerUniV2 = async (zetaToken_: string, uniswapV2Router_: string) => {
+  const Factory = (await ethers.getContractFactory("ZetaTokenConsumerUniV2")) as ZetaTokenConsumerUniV2__factory;
+
+  const ZetaTokenConsumerUniV2Contract = (await Factory.deploy(zetaToken_, uniswapV2Router_)) as ZetaTokenConsumerUniV2;
+
+  await ZetaTokenConsumerUniV2Contract.deployed();
+
+  return ZetaTokenConsumerUniV2Contract;
+};
```

Comment:
I think we can use `getContract` for this, like we did on the other deploys on top of this one.